packages/kbn-unified-data-table/src/utils/get_render_cell_popover.tsx

```diff
@@ -30,4 +32,10 @@ export const getCustomCellPopoverRenderer = () => {
 
     return <DefaultCellPopover {...props} />;
   });
+
+  // Component passed to the eui must be a non-memoized component
+  // otherwise eui throws an error `typeof Component !== 'function'`
```

Comment:
```suggestion
  // Components passed to EUI cannot be memoized components
  // otherwise EUI throws an error `typeof Component !== 'function'`
```

💅

Reply:
done: f4645febed4